test/fixtures/criteria.yml

```diff
@@ -0,0 +1,41 @@
+valid_criterium:
```

Comment:
i don't think so, because of rails conventions.

table names are plural (criteria), and class names are singular (criterium). i started out with the class name of criteria and couldn't get test fixture associations to run at all. rails just assumes criterium live in the criteria table, and uses that to work out association names and everything else. we could specify that criteria is singular and criterias is the table name, but I don't really like that either 

because it's just the name of an example record here it could be anything, but to keep it consistent with an individual record being a criterium, i'd leave it like this.

Reply:
Ah yeah it does makes sense - I wasn't sure since lines 29 and 36 are plural, but I suppose those are relating to array attributes on a criterium record whereas this is one criterium record (with attributes that are themselves criteria 🤔)